source/scriptHandler.py

```diff
@@ -290,7 +290,9 @@ def executeScript(script, gesture):
 	try:
 		scriptTime = time.time()
 		scriptRef = weakref.ref(scriptFunc)
-		if (scriptTime - _lastScriptTime) <= 0.5 and scriptFunc == lastScriptRef:
+		if (scriptTime - _lastScriptTime) * 1000 <= config.conf["keyboard"][
```

Comment:
Doesn't this break preferred linting, by putting the `and` on the same line as part of the condition, even if only a closing bracket?

Because I don't know how to accessibly do a multi line replacement suggestion, here are three suggestions achieving what I am thinking, the last two just deleting lines. Applying them as a batch should be possible, if you agree.

```suggestion
		if (
			(scriptTime - _lastScriptTime) * 1000 <= config.conf["keyboard"]["multiPressTimeout"]
			and scriptFunc == lastScriptRef
		):
```

Reply:
What do you call "preferred linting".

What I had done was not very readable; maybe it was automatically formatted by the linter by the way.
In any case, your proposal is definitely more readable. Thanks.

Reply:
> In any case, your proposal is definitely more readable. Thanks.

But apparently not acceptable to the linter? Interesting.

Reply:
Unfortunately, the linter has complained and I have had to restore the code as formatted by Ruff.

As @XLTechie has pointed out, the code as formatted by Ruff is less readable than being able to put each condition on a separate line, i.e.:
```
if (
	condition1
	and condition2
):
```

The only way I have found to force Ruff to keep this formatting (except a noqa that I have not tested) is to add comments between each clause:
if (
	# A comment explaining condition 1
	condition1
	# A comment explaining condition 2
	and condition2
):
```


Though, the code seemed quite self explaining to me, especially when there is a condition by line, and I did not want to overload it with comments not required for clarity.

It seems to me that this specific case is a case where Ruff linter made the code less readable with respect to what we had before. Or is there a way to force line splitting more logically on such if condition? I mean without noqa, nor comments.

Cc @seanbudd, @LeonarddeR for opinion.